aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -2569,6 +2571,98 @@ struct KronImpl final {
 };
 }
 
+DEFINE_DISPATCH(unpack_pivots_stub);
+
+std::tuple<Tensor, Tensor, Tensor> lu_unpack(
+    const Tensor& LU_data,
+    const Tensor& LU_pivots,
+    bool unpack_data,
+    bool unpack_pivots
+    ) {
+  TORCH_CHECK(LU_pivots.is_contiguous() && (LU_pivots.scalar_type() == at::kInt),
+      "LU_pivots is expected to be a contiguous tensor of torch.int32 dtype."
```

Comment:
nit: start error messages with the name of the user-facing function that throws them, like "lu_unpack: expected LU_pivots to be a contiguous tensor with torch.int32 dtype"